.github/workflows/ci.generate.ts

```diff
@@ -343,6 +349,11 @@ const ci = {
             job: "test",
             profile: "release",
             skip_pr: true,
+          }, {
+            os: Runners.macosArm,
+            job: "test",
+            profile: "release",
+            skip_pr: true,
```

Comment:
Do we have a way to prevent this from running on main as well and only run on tags?

Reply:
I think it would be possible to add a parallel option to skip_pr, but I wouldn't mind running this on main a few times just to make sure it actually works. :)

Reply:
@dsherret I think I have it working now.